### PR TITLE
Replaced Lee's Photos with Five images

### PR DESCRIPTION
**Pull request by nwanduka**
Replaced Lee's photos with images of Five in the [community page](https://layer5.io/community)

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 01cf51cd65f320c4edc3b28bd46d4f45a5b02fdd at: https://65933108bae3c006fb8ff4af--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3e4bcd1d2f366a2cb89ae8a85e7dd878ee9f321b at: https://659568cedbb1c80098cbc205--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c75ee74b09946f73a663b2af1130d4806cef5ca3 at: https://65956c33b8ce00032852c010--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4c4a9b3497f97da905caebc410cd699f7f6ea8df at: https://65973875993dc54ad984325b--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 17a212aed12d4fce74ee1954da6a75b6ff61c22c at: https://65974118a9f3f6504367fd5a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3156c89346bd57c92908f918587d8a47fb8650de at: https://6597457d2209f14e6ace15ff--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2e57e445931fa05566772ffb030001277d2f4f06 at: https://659be30f4804834fd7fabd03--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1513c8d754020f2c4f99e584167bf8afa74313dd at: https://659bea414cfc79524c7e8a20--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b8eb698e5dbe9836cd9184f6f3dc82612cf1b143 at: https://659bf0c2f50dbf144c03c544--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 087f3c5fc476fb361e014f25a02d775e248be6b0 at: https://659c0a4da5c08502dc3ed1c4--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 242b2c7e7bb9c49864076572884c6c87b160ee90 at: https://659c0da0de69a40ebd354f82--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7c27b4d3cd6fb12a3d654aefeb2ace5e30595c7a at: https://65a29ba9016b53c989d06c7d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 078da644f079c6fdbad3a7986decf9053587e3b2 at: https://65a2b150389b9cd51b9b5355--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b11cac48f03a03ebf501ad1ce5ef997cec4d33a5 at: https://65a2bfb199d609e256b26ca3--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit aa8d3a76018b583c0486d010a8fd3ec76bba2f00 at: https://65a520607f41e739cd2d31b4--layer5.netlify.app

File: src/sections/Community/Community-pictures/14.png
Comment (leecalcote) [inline src/sections/Community/Community-pictures/14.png:1]: Five shouldn't be covered in color.

**Comment by l5io** [comment]
🚀 Preview for commit d6ad8a589269812797828892b09a88504e85782c at: https://65c0c74463853f06a39cdc26--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 55ff7412d0c732a8ba48db0fb50905d0564416f3 at: https://65c0d329d4ea7d0e032efeeb--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a5f9d5ca6057cfc685d2fbd9544738051686f80f at: https://65c0dd3ab16a0111e8852ec7--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8f1adfa8e958e34abd55555b18176306002f497c at: https://65c0e19f63853f19f19cddd2--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 29479a7ce1f076a0779a75e960a0afe6fac2315b at: https://65c0e6e81f80cc1de3142a9a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7a8be737f30671824d579b25ecfa4ccd3c3718a0 at: https://65c0f33e13ac5425d8abf620--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7261c4b34e344b7110cc1fb5ebd301faa39f5ccb at: https://65c0fb1e5707f52dc7ff750e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a863be21d2e993a78f68a2fe38af8564fa43d0a2 at: https://65c521921047a80399e56ad7--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d8074f5a9dd6581a2c9dbcfb4e35925d47dc4cce at: https://65ca1db0a108d40343512e41--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c3023e23e51cda39823318a450f4df46dca2ccc9 at: https://65ca2b09722bc9f0b26e67df--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 94d66d6b55fa6d3a0d2d01e91cf8c3950ab99c96 at: https://65cb9a5752aaf407f7fa2681--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b2b29faf6dca4e366c64df677ccd8e69c10d50da at: https://65ce34bbb4ffcc057355ff1b--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7ed3114666330cc7ef3000a1e409c196dc227d8a at: https://65ce3878c327a30833198313--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b24e05b303cc1422bdca43f18eae153064d84f77 at: https://65d2417136f61e7795f0b923--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0d3e629b2d9c1ab4196962eb764e9a595c80cd09 at: https://65d247eabb272e7e182f836e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e0a3a24361b43d5d6231178861c2aa40807d8699 at: https://65d34686041c3b00872a4080--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 90cadad2c8b85d530e376dd196309578e58485ca at: https://65d356442e81520fa4c57498--layer5.netlify.app

File: src/sections/Careers/index.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ import { Container } from "../../reusecore/Layout";
 import { Row } from "../../reusecore/Layout";
 import PageHeader from "../../reusecore/PageHeader";
 import InternshipPage from "./Careers-Internship-grid";
-import Lee_giveaway from "../../assets/images/careers/Lee-giving-away-free-t-shirts-at-UT-Austin.jpeg";
+import Five from "../../sections/Community/Community-pictures/14.svg";
 import JoinCommunity from "../Community/Join-community";
 import CareersSectionWrapper from "./careers.style";
 
-const Lee_and_Rigs = "../../assets/images/careers/Lee-and-Rigs-Caballero-of-Google.jpeg";
+const Five = "../../assets/images/careers/Five.svg";
```
[the remark this answers is not in the document]
Comment (nwanduka) [reply src/sections/Careers/index.js:11]: Ohh.. okay. Thanks for pointing that out.

**Comment by l5io** [comment]
🚀 Preview for commit 58edef607ae7fcf38d2e68006097619f31795b58 at: https://65d654bf0c8e58fbd5c5db52--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 300d51b7dfb81fff0fbc34638b76999cfd6f44e0 at: https://65f82d25d3f0445a91eb97ff--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3313361501d24334d1ac48c483050b17712f7fe5 at: https://65f83d702115b967d31a0004--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ef0d45797c1800972560253923a20532ec97f9e1 at: https://65f845780df21d6c21ec2d51--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ab0f61cbb16650468a12bd189b473505d8f34f6b at: https://65f84e0cc265726fc8aa249b--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 387ccd03d56aafb5221297f26ef46d120b54189f at: https://65f853491487587998079f6c--layer5.netlify.app

**Comment by iArchitSharma** [comment]
@nwanduka are you done here?

**Comment by leecalcote** [comment]
Merge conflicts

**Comment by ritiksaxena124** [comment]
This item is completed by another PR which is merged. So closing this. Thank you @nwanduka 🙌 for taking this.

**Pull request closed without merging**